distro/artix.yaml

```diff
@@ -0,0 +1,28 @@
+name: Artix Linux
+os-type: GNU/Linux
+homepage: https://artixlinux.org/
+based-on:
+  - Arch Linux
+  - Arch OpenRC
+  - Manjaro OpenRC
+origin:
```

Comment:
No specific founder / founding location (at least I didn't find any) since it got created merging Arch OpenRC and Manjaro OpenRC

Reply:
hmm